applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -92,7 +93,7 @@ static void connected_cb(struct bt_conn *conn, uint8_t err)
 		}
 
 		if (IS_ENABLED(CONFIG_BT_PERIPHERAL)) {
-			/* Start adv? */
+			/* TODO: Start adv? */
 		}
```

Comment:
Should either start adv or remove this code

Reply:
Removed the TODO and added bt_mgmt_adv_restart()